README.md

```diff
@@ -1320,6 +1379,15 @@ __AWESOME__ apps counter: __150__ 🎉
 - [x] [GitHub](https://github.com/Dimowner/AudioRecorder)
 - [ ] Official page
 
+## VPN
```

Comment:
Yeah I was very hesitant with VPNs, because to be honest the paid are just the best comparing to free ones.

Reply:
I know how VPNs works, however the `## VPN` section would be `Free of charge` and well compared to the paid ones I don't know about this.

Reply:
Paid one can not be included in the list, so should I drop `## VPN` category?

Reply:
To be honest I don't know. Free VPNs to be are more like marketing thing than real thing.

Reply:
Calyx VPN, riseup VPN based on bitmask aren't a marketing thing?